tasks/main.yml

```diff
@@ -3,34 +3,58 @@
   fail: msg="ansistrano_custom_tasks_path is not used anymore. Please, check the documentation at https://github.com/ansistrano/deploy"
   when: ansistrano_custom_tasks_path is defined
 
-- name: ANSISTRANO | Execute "Before Setup" Tasks
-  include: "{{ ansistrano_before_setup_tasks_file | default('empty.yml') }}"
+- include: "{{
+    ansistrano_before_setup_tasks_file
+    | default(ansistrano_display_default_hooks is defined and ansistrano_display_default_hooks)
```

Comment:
Ansible implementation note: it IS possible to make it work with default(var) __but__ only if that var is specified in `vars` __of the executing role__. (`vars` come into scope earlier than `group_vars`, i.e. they are not host-specific) But ansistrano builds on the premise that playbooks can override role vars, it is not an applicable solution: vars in the role can never be overridden in time from the playbook. 

Not sure whether that is a bug or feature.